randovania/games/samus_returns/exporter/joke_hints.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+JOKE_HINTS = [
+    "This hint has been sealed away.",
+    "Spidersparking can quickly cover large distances.\nTake advantage whenever possible!",
+    "The Spider Boost can be used while in Spider Ball form.\n"
+    "It will launch you forward, damaging enemies you make contact with.",
+    "Drag pins onto the map to mark points of interest.",
+    "This hint hasn't returned yet.",
+    "Walls are just a suggestion.",
+    "See you next mission!",
+    "In the year 20X5 of the Cosmic Calendar...",
+    "The resulting decision of the Galactic Federation Council was immediate and unanimous.",
+    "Samus Aran... You must exterminate the Metroids once and for all!",
+    "A Galactic Federation research team discovered an unknown life-form on planet SR388.\n\n"
+    "While they were able to successfully obtain a living sample, "
+    "their research vessel was attacked by Space Pirates during their voyage home.",
+    "This attack was no coincidence.\n"
+    "The Space Pirates had set their sights on these mysterious organisms called Metroids.",
+    "They planned to replicate the sample and exploit its ability to absorb the energy of any life-form. "
+    "Their intention was to use Metroids as a weapon, one powerful enough to conquer the entire galaxy.",
+    "To combat this threat, the Galactic Federation dispatched a lone bounty hunter "
+    "to infiltrate the Space Pirate base on the planet Zebes.",
+    "Against all odds, Samus Aran eradicated the Metroids on Zebes and defeated Mother Brain, "
+    "the leader of the Space Pirates. And so their sinister plans were thwarted... for a time.",
+    "Concerned by these developments and by the great threat the Metroid species still posed to the galaxy, "
+    "the Galactic Federation mounted another expedition to SR388.",
+    "A special squadron of elite soldiers from the Galactic Federation Police was dispatched to investigate.\n\n"
+    "The team soon went missing, but not before transmitting a small sampling of data back to headquarters.",
+    "Analysis of this data confirmed a Metroid presence within the interior of planet SR388. "
+    "So long as they continued to exist, these Metroids would forever pose a devastating threat to the galaxy.",
```

Comment:
Is this supposed to be a giant multiline hint?

Reply:
Yes. It just fits in the textbox.

Reply:
oh these are multiple hints, i didnt see it at first. Eh, personally not a fan to stuff that amount of in-game text as a hint, as it increases the chances of people thinking that their game bugged out.

Reply:
I could spit the ones with multiple sentences into separate joke hints Or add `...` on either side and cut off a much of the text to make it feel like it's a joke hint.

Reply:
I'd def change the multi-line hints a lot to make them feel more jokey.